packages/cli/src/module-resolver.ts

```diff
@@ -339,5 +358,10 @@ export function getCommonJsExports(
   }
 
   const { exports, reexports } = parse(code);
-  return [...exports, ...reexports];
+
+  // Re-exports are paths to exports that must be resolved themselves
+  const resolvedReexports = reexports.flatMap((reexport: string) =>
```

Comment:
Should we account for export cycles here? Not sure if those realistically will occur